src/components/grapher/Grapher.vue

```diff
@@ -199,5 +216,7 @@ export default Vue.extend({
 
 .grapher--field-rect {
   fill: $soybean;
+  stroke: $white;
```

Comment:
Instead of this, how about we combine with the CSS rule for `.grapher--line-container`?

You can select multiple items with a comma, for example, on line 200, change from 
```css
.grapher--line-container {
```
to
```css
.grapher--line-container, .grapher--field-rect {
```

Reply:
Good idea.